docs/Contributing/API-for-contributors.md

```diff
@@ -2556,7 +2556,17 @@ Returns the host information about the device that makes the request.
         "cycle_count": 999,
         "health": "Good"
       }
-    ]
+    ],
+    "mdm": {
+      "encryption_key_available": false,
+      "enrollment_status": null,
+      "name": "",
+      "server_url": null,
+      "macos_settings": {
+        "disk_encryption": null,
+        "action_required": null
+      }
```

Comment:
Martin I think we're missing documentation here for the `mdm.profiles` field, as well for the `GET host` response in REST-API.md (line 2249) – would you mind adding those in this PR?

The typing from the front end is `IMacSettings` below (does this match the API types?):
<img width="511" alt="Screenshot 2023-03-08 at 12 11 30 PM" src="https://user-images.githubusercontent.com/61553566/223837954-b7871e1d-75d6-43dc-ae0b-82d8eb70b470.png">

and an example response:
<img width="516" alt="Screenshot 2023-03-08 at 12 10 18 PM" src="https://user-images.githubusercontent.com/61553566/223837495-8fda35cc-ff0a-4635-aeab-10af436e1c3f.png">

Reply:
@jacobshandling Good catch, I have to work on a doc-only PR for a bunch of small fixes, I'll add those as part of that other PR.